singularity/debian/singularityfile_debian

```diff
@@ -0,0 +1,149 @@
+Bootstrap: docker
+From: debian:bullseye
+
+%help
+Singularity container for GRASS GIS to be run into GRASS main directory
+
+%labels
+  Maintainer Luca Delucchi
+
+%setup
+  mkdir /tmp/grass_build
+
+%files
+  . /tmp/grass_build
+
+%post
+    # Install useful libraries
+    apt-get -y update
+    apt-get -y install \
+        build-essential \
+        bison \
+        bzip2 \
+        cmake \
+        curl \
+        flex \
+        g++ \
+        gcc \
+        gdal-bin \
+        gettext \
+        git \
+        libbz2-dev \
+        libcairo2 \
+        libcairo2-dev \
+        libcurl4-gnutls-dev \
+        libfftw3-bin \
+        libfftw3-dev \
+        libfreetype6-dev \
+        libgdal-dev \
+        libgeos-dev \
+        libgsl-dev \
+        libjpeg-dev \
+        libjsoncpp-dev \
+        libopenblas-base \
+        libopenblas-dev \
+        libnetcdf-dev \
+        libncurses5-dev \
+        libopenjp2-7 \
+        libopenjp2-7-dev \
+        libpdal-dev \
+        libpnglite-dev \
+        libpq-dev \
+        libpython3-all-dev \
+        libreadline-dev \
+        libsqlite3-dev \
+        libtiff-dev \
+        libzstd-dev \
+        make \
+        mesa-common-dev \
+        moreutils \
+        ncurses-bin \
+        netcdf-bin \
+        pdal \
+        python3 \
+        python3-dateutil \
+        python3-dev \
+        python3-gdal \
+        python3-magic \
+        python3-numpy \
+        python3-pdal \
+        python3-pil \
+        python3-pip \
+        python3-ply \
+        python3-requests \
```

Comment:
AFAIK python3-requests is not used in GRASS GIS

Reply:
But it is required from some addons (i.modis and also i.sentinel) that the script install.